sdk/python/pvsite_datamodel/sqlmodels.py

```diff
@@ -38,27 +36,33 @@ class SiteSQL(Base, CreatedMixin):
 
     site_uuid = sa.Column(UUID(as_uuid=True), default=uuid.uuid4, primary_key=True)
     client_uuid = sa.Column(
-        UUID(as_uuid=True), sa.ForeignKey("clients.client_uuid"), nullable=False, default=uuid.uuid4
+        UUID(as_uuid=True),
+        sa.ForeignKey("clients.client_uuid"),
+        nullable=False,
     )
-    client_site_id = sa.Column(INTEGER)
-    client_site_name = sa.Column(sa.String(255))
+    client_site_id = sa.Column(sa.Integer, index=True)
+    client_site_name = sa.Column(sa.String(255), index=True)
+
     region = sa.Column(sa.String(255))
     dno = sa.Column(sa.String(255))
     gsp = sa.Column(sa.String(255))
-    orientation = sa.Column(REAL)
-    tilt = sa.Column(REAL)
-    latitude = sa.Column(FLOAT, nullable=False)
-    longitude = sa.Column(FLOAT, nullable=False)
-    capacity_kw = sa.Column(REAL, nullable=False)
-    updated_utc = sa.Column(TIMESTAMP, nullable=False, default=lambda: datetime.utcnow())
```

Comment:
Note that I removed `updated_utc` for now. It wasn't used properly anyway. If we want it we should put it back with a proper postgreql trigger so that it is automatically updated.